src/components/Examples/ExamplesGrid.tsx

```diff
@@ -77,26 +79,36 @@ const ExamplesGrid = ({
 
   return (
     <div className="grid grid-cols-[repeat(auto-fill,_minmax(260px,_1fr))] gap-x-20 gap-y-32">
-      {examples.map(({ name, description, languages, products, useCases, image }, key) => (
-        <div className="w-full relative overflow-hidden group/examples-index-card" key={`${name}-${key}`}>
+      {examples.map(({ id, name, description, languages, products, useCases }, key) => (
+        <div
+          onClick={() => navigate(`/examples/${id}`)}
```

Comment:
[nitpick] This is what I meant in standup, would crawlers be able to navigate to the example as well.

Reply:
Let's do it, I've pushed it up, don't need any of the extra a11y attributes either as it's a better semantic fit. Perhaps this was a "can't nest an X inside a Y error" from a past point where it had a different structure.